packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -2998,6 +3006,11 @@ export interface SystemMessageCommon extends MessageCommon {
     messageType: 'system';
 }
 
+// @beta
+export type TeamsAdapterOptions = {
+    onFetchProfile?: OnFetchProfileCallback;
```

Comment:
should this be a required param? Otherwise we don't display the current users info

Reply:
This is a good point - I got the same problem and discussed that a bit, according to previous discussions:
1. CTE can work with Teams client (especially 1 to 1, or 1 to n) without this one
2. Displayname could be potentially coming from sdk in the future
So the discussion result is not to make this required, to make it easier to use for one CTE to multiple teams client scenarios and leave the possibility for displayname coming into sdk